### PR TITLE
add spec for inline localization

### PR DESCRIPTION
**Pull request by RWOverdijk**
Tried sticking to the code style (double quotes and odd comma placement and such). 😄

### CODE REVIEW

**Comment by PikachuEXE** [comment]
Thanks!
I will try to implement it tomorrow or next week...
You can try to implement it too 😄

**Comment by RWOverdijk** [comment]
@PikachuEXE I gave it a go and update the PR. Basically I just pass on the options to the lookup.

File: app/assets/javascripts/i18n.js [hunk 1]
```diff
@@ -799,17 +799,17 @@
...
       case "currency":
-        return this.toCurrency(value);
+        return this.toCurrency(value, options);
       case "number":
-        scope = this.lookup("number.format");
+        scope = this.lookup("number.format", options);
         return this.toNumber(value, scope);
-      case "percentage":
-        return this.toPercentage(value);
+        case "percentage":
```
Comment (PikachuEXE) [inline app/assets/javascripts/i18n.js:806]: Why is these 2 lines indented?

Comment (RWOverdijk) [reply app/assets/javascripts/i18n.js:806]: @PikachuEXE Because I made a mistake 😄

**Comment by PikachuEXE** [comment]
> refix indentation
2nd line also indented (`return this.toPercentage(value, options);`)

**Comment by PikachuEXE** [comment]
And please also [squash the commits](https://gist.github.com/patik/b8a9dc5cd356f9f6f980)

**Comment by RWOverdijk** [comment]
GitHub can do that nowadays I think?

**Comment by RWOverdijk** [comment]
Anyway, done.

**Pull request closed without merging**